src/screens/GrantInfoScreen/GrantInfo.tsx

```diff
@@ -1,19 +1,69 @@
-import { Text, View } from 'react-native';
-import globalStyles from '../../globalStyles';
+import { Text, View, ScrollView, Image } from 'react-native';
+import Icon from '../../../assets/icons';
 import { GrantsStackScreensProps } from '../../types/navigation';
 import styles from './styles';
+import RectButton from '../../components/RectButton';
+import grantPlaceholder from '../../../assets/grantPlaceholder.png';
 
 function GrantInfoScreen({ route }: GrantsStackScreensProps<'GrantInfo'>) {
   const { grantObj } = route.params;
   return (
-    <View style={styles.container}>
-      <Text style={globalStyles.h2}>{grantObj.title}</Text>
-      <Text>Amount: {`$${grantObj.amount.toString()}`}</Text>
-      <Text>Eligible Countries: {grantObj.countries.join(', ')}</Text>
-      <Text>Brief: {grantObj.description}</Text>
-      <Text>Duration: {grantObj.duration}</Text>
-      <Text>Deadline: {new Date(grantObj.deadline).toLocaleDateString()}</Text>
-    </View>
+    <ScrollView style={styles.container}>
+      <View style={styles.backArrow}>
+        <Icon type="back_arrow" />
+      </View>
+      <View style={styles.innerContainer}>
+        <View>
+          {/* might have to handle font size for longer titles */}
+          <Text style={styles.titleText}>{grantObj.title}</Text>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={grantPlaceholder} style={styles.image} />
+        </View>
+        <RectButton
```

Comment:
rect button seems to automatically capitalize text, don't know if this should be a problem

Reply:
Oh yeah you're right! I just used the RectButton that we've been using but I can make another button component if you want.